js/tic_tac_toe.js

```diff
@@ -1,2 +1,80 @@
-// Your code here
-// Use the jQuery
+$(document).ready(function() {
+  $('.box_cell').on('click', TicTacToe.takeTurn);
+});
+
+
+var TicTacToe = (function(){
+  var turnCounter = 1;
+  var board = [
+    [0,0,0],
+    [0,0,0],
+    [0,0,0]
+  ]
+
+  var takeTurn = function() {
+    if (!$(this).hasClass('used')){
+        turnCounter % 2 === 0 ? redTurn($(this)) : blackTurn($(this));
+        $(this).addClass('used');
+        turnCounter++;
+        checkBoard();
+      }
+  };
+
+  var redTurn = function(cell){
+    var row = cell.data().row;
+    var col = cell.data().col;
+    board[row][col] = 1;
+    cell.css('background-color', 'red');
+  };
+
+  var blackTurn = function(cell){
+    var row = cell.data().row;
+    var col = cell.data().col;
+    board[row][col] = -1;
+    cell.css('background-color', 'black');
+  };
+
+  var checkRows = function() {
+    for (var i = 0; i < 2; i++) {
+      var total = board[i].reduce(sum);
```

Comment:
Good use of reduce, but you'll want to remove the definition of `total` from the for loop, as its being initialized 3 times.